### PR TITLE
Release v10.5 azure

### PR DESCRIPTION
**Pull request by zivshits**
- Add customer network
- Update import-pas-images script to handle single components
- Fix import-pas-images script to be error prone

### CODE REVIEW

File: azure/pas-single-component-deploy-in-customer-network.json [hunk 1]
```diff
@@ -0,0 +1,289 @@
...
+        "License Agreement": {
+            "defaultValue": "Decline",
+            "allowedValues": [
+                "Accept",
+                "Decline"
+            ],
+            "maxLength": 6,
+            "type": "String",
+            "metadata": {
+                "description": "I have read and agree to the Terms and Conditions."
```
Comment (nimrody15) [inline azure/pas-single-component-deploy-in-customer-network.json]: I have read and agreed to the Terms and Conditions.

File: azure/pas-single-component-deploy-in-customer-network.json [hunk 2]
```diff
@@ -0,0 +1,289 @@
...
+            }
+        },
+        "Vault Private IP": {
+            "type": "String",
+            "metadata": {
+                "description": "Enter the IP of the Vault instance and the DR IP (if exists) seperated by a coma (e.g 1.1.1.1,2.2.2.2)."
```
Comment (nimrody15) [inline azure/pas-single-component-deploy-in-customer-network.json]: comma

File: azure/pas-single-component-deploy-in-customer-network.json (hunk 1 above)
Comment (zivshits) [reply azure/pas-single-component-deploy-in-customer-network.json]: Done

File: azure/pas-single-component-deploy-in-customer-network.json (hunk 2 above)
Comment (zivshits) [reply azure/pas-single-component-deploy-in-customer-network.json]: Done

**Pull request merged**